src/app/batch_generate_keys/batch_generate_keys.ml

```diff
@@ -159,6 +161,107 @@ let make_graphql_signed_transaction ~sender_priv_key ~receiver ~amount ~fee =
      application/json' --data '%s'@."
     (Yojson.Basic.to_string graphql_query_json)
 
+let there_and_back_again ~count ~txns_per_block ~slot_time ~fill_rate
```

Comment:
Also, add a final `()` argument here to appease the typechecking gods

Reply:
🕯️